Cirrious.FluentLayout/FluentLayoutExtensions.cs

```diff
@@ -4,6 +4,9 @@
 // Contributions and inspirations noted in readme.md and license.txt
 // 
 // Project Lead - Stuart Lodge, @slodge, me@slodge.com
+//
+//Softlion: SubviewsDoNotTranslateAutoresizingMaskIntoConstraints can now  go through all levels recursively
```

Comment:
in general, i **try** to add comments for 3 reasons...

1. legal - although I'm slowly trying to get rid of those headers
2. attribution - this is partly legal, but more about being nice when saying when code or ideas have been borrowed/inspired
3. when code needs it for readability or explanation (e.g. if a "+1" is needed in a loop somewhere I might explain why...)

Comments like 
```
/// <summary>
/// Adds two numbers together
/// </summary>
/// ...
public double Add(...
```
impress me when they are added everywhere and maintained to a high level, but otherwise don't :)

That's just me - IMO - I know others who refuse to use a library if it doesn't have `<summary>` tags! #liveandLetLive :)